src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/Instrument.cs

```diff
@@ -0,0 +1,151 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Diagnostics.Metrics
+{
+    /// <summary>
+    /// Base class of all Metrics Instrument classes
+    /// </summary>
+#if ALLOW_PARTIALLY_TRUSTED_CALLERS
+        [System.Security.SecuritySafeCriticalAttribute]
+#endif
+    public abstract class Instrument
+    {
+#if NO_ARRAY_EMPTY_SUPPORT
+        internal static KeyValuePair<string, object?>[] EmptyTags { get; } = new KeyValuePair<string, object?>[0];
+#else
+        internal static KeyValuePair<string, object?>[] EmptyTags { get; } = Array.Empty<KeyValuePair<string, object?>>();
```

Comment:
```C#
internal static KeyValuePair<string, object?>[] EmptyTags => Array.Empty<KeyValuePair<string, object?>>();
```
?

Reply:
what is the question?

Reply:
Whether we should change to the code I suggested (note the `=>` instead of the `{ get; } =`). i.e. we shouldn't need another dedicated backing field for the auto-prop as it's currently written.

Reply:
got it. thanks!